### ISSUE TITLE
Bad gateway error

### ISSUE THREAD

**Issue by KDurimel** [gristlabs/grist-omnibus#3]
Hello,

Firstly, I would like to extend my gratitude for your work. 

I am experiencing a "502 Bad Gateway" error while using Grist Omnibus and I am hoping you can assist me in resolving this issue.

I have tried several Docker invocations, including the simplest one provided in your [readme](https://github.com/gristlabs/grist-omnibus/blob/main/README.md). However, I am still encountering the error. Here is the simplest Docker command I used:

```
mkdir -p /tmp/grist-test
docker run \
  -p 9999:80 \
  -e URL=http://localhost:9999 \
  -e TEAM=cool-beans \
  -e EMAIL=owner@example.com \
  -e PASSWORD=topsecret \
  -v /tmp/grist-test:/persist \
  --name grist --rm \
  -it gristlabs/grist-omnibus  #
```

All ports on my server are open, and I even tried running the Docker command with the root user.

Could you please advise me on how to resolve this issue? All the services started successfully, but when I open the URL, I face a blank page with a "Bad Gateway" error. I would greatly appreciate any assistance you can provide.

Thank you for your time and attention.

**Comment by paulfitz** [comment]
I see an error on the latest image:
```
/grist/sandbox/run.sh: line 6: ./sandbox/gvisor/update_engine_checkpoint.sh: No such file or directory
```
Looks like something has broken in the build. Apologies, will look into it.

**Comment by paulfitz** [comment]
Hi @KDurimel, could you try again after repeating `docker pull gristlabs/grist-omnibus`? I made a fix in #4
